arrows/ocv/tests/CMakeLists.txt

```diff
@@ -9,10 +9,11 @@ set(test_libraries      vital vital_vpm kwiver_algo_ocv )
 ##############################
 kwiver_discover_tests(ocv_algo_config         test_libraries test_algo_config.cxx)
 kwiver_discover_tests(ocv_bounding_box        test_libraries test_bounding_box.cxx)
+kwiver_discover_tests(ocv_crop_chip           test_libraries test_crop_chips.cxx)
 kwiver_discover_tests(ocv_descriptor_set      test_libraries test_descriptor_set.cxx)
 kwiver_discover_tests(ocv_distortion          test_libraries test_distortion.cxx)
 kwiver_discover_gtests(ocv estimate_fundamental_matrix LIBRARIES ${test_libraries})
 kwiver_discover_gtests(ocv estimate_homography         LIBRARIES ${test_libraries})
 kwiver_discover_tests(ocv_feature_set         test_libraries test_feature_set.cxx)
 kwiver_discover_gtests(ocv image                       LIBRARIES ${test_libraries})
-kwiver_discover_tests(ocv_match_set           test_libraries test_match_set.cxx)
+kwiver_discover_tests(ocv_match_set           test_libraries test_match_set.cxx)
```

Comment:
Does this mean we lost or gained a new line at end of file? A new line at the end of file is always appreciated.

Reply:
Not sure exactly, but I can verify that a new line exists at the end of my current version of this file.